modules/web/gen-default-nodes/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import axios from 'axios';
+import fs from 'fs';
+import defToFragment from './def-to-fragment';
+
+/**
+ * parse fragment asynchronously
+ * @param {string} fragment - source fragment.
+ * @return {object} fragment details to be sent to fragment parser.
+ */
+
+const ENDPOINT = 'http://localhost:8289/ballerina/model/parse-fragment';
```

Comment:
Can we use https://github.com/ballerinalang/composer/blob/e70f81097763907c07aed2d7efc19fdb32aaa06d/modules/web/js/api-client/api-client.js#L34 to get the endpoint?

Reply:
we cannot import `api-client` here. Because we run this file in a command line environment. `api-client` has some webpack alias's (event_channel).